main.py

```diff
@@ -1,7 +1,47 @@
 import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = [
+        ("Western",),
+        ("Action",),
+        ("Drama",),
+    ]
+
+    actors = [
+        ("George", "Clooney"),
+        ("Keanu", "Reeves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson"),
+    ]
+
+    for (name,) in genres:
+        Genre.objects.create(name=name)
+
+    for first_name, last_name in actors:
+        Actor.objects.create(first_name=first_name, last_name=last_name)
+
+    Genre.objects.filter(name="Drama").update(name="Drama")
```

Comment:
I noticed that you're updating the name of the genre 'Drama' to 'Drama'. This operation seems redundant as it doesn't change the data. Consider removing this line if it's not needed.